### DISCUSSION TITLE
参数引用原文件可否使用之前onedrive-cf-index的样式

### DISCUSSION THREAD

**Discussion by Huiaei** [spencerwooo/onedrive-vercel-index#39] [General]
之前onedrive-cf-index的方式是直接在链接后面加`?raw`
我注意到vercel的这个版本是  **`域名`/api?path=`文件路径`&raw=true**
这导致我之前按照`?raw`方式作为的图床链接全部失效，而且也不好直接让之前Cloudflare的域名直接用重定向的方式到vercel域名来使之前的外链图重新生效。

**Comment by spencerwooo** [comment]
Vercel 版本的文件直链是在 Serverless 部分实现的，跟原版有较大的不同，沿用原版链接格式实现极为不方便，估计是不会向前兼容了。（另外原版也不是不能用。如果替换起来实在麻烦其实原版依旧可以留着，换个域名部署新版。）

**Reply by Huiaei** [reply]
了解了，既然不会向前兼容，那我还是得慢慢的向新版跟进。
不知道cf worker能不能实现自动替换url的操作。
